appsettings.Development.json

```diff
@@ -4,5 +4,10 @@
       "Default": "Information",
       "Microsoft.AspNetCore": "Warning"
     }
-  }
+  },
+  "MongoDbSettings": {
+    "ConnectionString": "mongodb+srv://aliakseidziatlau:Alexey200616may@officeservicecluster.0dsu1.mongodb.net/?retryWrites=true&w=majority&appName=OfficeServiceCluster",
```

Comment:
usually such data is not sent to the repository, it can be saved in user secrets.